homeassistant/helpers/script.py

```diff
@@ -274,7 +274,7 @@ async def async_validate_action_config(
         )
 
     elif action_type == cv.SCRIPT_ACTION_REPEAT:
-        config[CONF_SEQUENCE] = await async_validate_actions_config(
+        config[CONF_REPEAT][CONF_SEQUENCE] = await async_validate_actions_config(
```

Comment:
Wow. How could this ever work ? 

Please add a test

Reply:
uh

Reply:
Looking at it again, I see now that we still have the original automation config in `config[CONF_REPEAT][CONF_SEQUENCE]` but no validated stuff, guess that could work in some cases.